Classes/common/HTTP/CDTURLSessionInterceptorContext.h

```diff
@@ -0,0 +1,39 @@
+//
+//  CDTURLSEssionFilterContext.h
+//  
+//
+//  Created by Rhys Short on 17/08/2015.
+//
+//
+
+#import <Foundation/Foundation.h>
+#import "CDTMacros.h"
+
+@interface CDTURLSessionInterceptorContext : NSObject
+
+@property (nonnull, readonly, nonatomic, strong) NSMutableURLRequest *request;
```

Comment:
After the talk on slack, I've added a note to the read in commit 98698c1 to explain that we only support building on the latest stable Xcode.